test/presence.js

```diff
@@ -104,7 +104,11 @@ describe('Presence', function() {
 					results.push([e.watcher_count, e.user.id]);
 					// expect to see thierry join, james join and james leave
 					if (results.length === 3) {
-						const expected = [[1, 'user1'], [2, 'james'], [1, 'james']];
+						const expected = [
+							[1, 'user1'],
+							[2, 'james'],
+							[1, 'james'],
+						];
```

Comment:
why do we have this diff?

Reply:
Its a lint fix

Reply:
Its a lint fix

Reply:
it's not - old version was fine. 
https://travis-ci.com/GetStream/stream-chat-js/jobs/261569670#L227

check and update your local prettier version.